src/server/scripts/Pet/pet_generic.cpp

```diff
@@ -320,10 +320,30 @@ class npc_pet_gen_mojo : public CreatureScript
         }
 };
 
+enum SoulTrader
+{
+    SAY_INRO                        = 0,
```

Comment:
Maybe I am too nit-picky here, but shouldn't it be `SAY_INTRO` instead of `SAY_INRO` ?
I googled "inro" to see if it had a useful meaning, but none of them seem related.

Reply:
Fixed. Thanks :)